src/gui/fspathedit_p.cpp

```diff
@@ -251,8 +251,7 @@ void Private::FileLineEdit::keyPressEvent(QKeyEvent *e)
         showCompletionPopup();
     }
 
-    const FileSystemPathValidator *validator =
-        qobject_cast<const FileSystemPathValidator *>(this->validator());
+    const auto *validator = qobject_cast<const FileSystemPathValidator *>(this->validator());
```

Comment:
Sorry, I lost your previous comment, so I can't quote it... But I guess you didn't know what I meant.
Since you have `qobject_cast<const FileSystemPathValidator *>` at right hand both `const auto *validator` and `auto *validator` (and even `auto validator`) are deduced to `const FileSystemPathValidator *` so `const` at left is redundant!

Reply:
And i was wondering where this comment went. Thanks for the explanation, i'll remove it.